elasticdl/python/master/pod_manager.py

```diff
@@ -430,15 +420,28 @@ def _event_cb(self, event):
             if pod_name in self._failed_pods:
                 return
 
+            # Notify each PodEventCallback that PodStarted is fired
+            if evt_type in ["ADDED", "MODIFIED"] and phase == "Running":
```

Comment:
`ADDED` can be removed here, as a pod is always MODIFIED from pending to running. `ADDED` with running only happens without pod status change.
Since we will track pod status change later, this can leave to the next PR.

Reply:
If the image is cached in the Node, we can see ADDED with phase `RUNNING` directly.